server/src-lib/Hasura/Eventing/ScheduledTrigger.hs

```diff
@@ -404,18 +459,33 @@ processStandAloneEvents logger logEnv httpMgr pgpool = do
   where
     logInternalError err = L.unLogger logger $ ScheduledTriggerInternalErr err
 
+    saveLockedStandAloneEvents :: [StandAloneScheduledEvent] -> IO ()
+    saveLockedStandAloneEvents standAloneEvents =
+      liftIO $ atomically $ do
```

Comment:
No need to use `liftIO` here if you’re just in plain old `IO`—`liftIO` is only necessary if you are in some other monad that wraps `IO`.